lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -122,40 +115,46 @@ const rule = (primary, _secondaryOptions, context) => {
 		}
 
 		root.walkDecls((decl) => {
-			if (!isStandardSyntaxDeclaration(decl) || !isStandardSyntaxProperty(decl.prop)) {
-				return;
-			}
+			if (!isStandardSyntaxDeclaration(decl)) return;
 
-			const prop = decl.prop;
-			const value = decl.value;
+			const { prop, value } = decl;
+
+			if (!isStandardSyntaxProperty(prop)) return;
+
+			if (!isStandardSyntaxValue(value)) return;
 
 			const normalizedProp = vendor.unprefixed(prop.toLowerCase());
 
-			if (hasIgnoredCharacters(value) || !isShorthandProperty(normalizedProp)) {
-				return;
-			}
+			if (!isShorthandProperty(normalizedProp)) return;
```

Comment:
nit

rename to `isSupportedShorthand`
else we would have used `longhandSubPropertiesOfShorthandProperties.keys()`

Reply:
Make sense. Done b290aee10